src/models.ts

```diff
@@ -0,0 +1,24 @@
+export class Resource {
+	name: string;
+	key: string;
+	tags: Array<string>;
+}
+
+export class Flag extends Resource {
+	environments: Map<String, Environment>;
+}
+
+export class Environment extends Resource {
+	_site: { href: string };
+}
+
+export class FlagConfiguration {
+	key: string;
+	variations: Array<any>;
```

Comment:
It'd be nice if we could get more specific types here. I have a feeling we can import this type from https://github.com/launchdarkly/api-client-typescript but I haven't tried it yet

Reply:
I was hoping to use api-client-typescript for both the API and types, but unfortunately we haven't released it as an npm package yet. I'll ask around to see if we can get that out.